resources/com/soywiz/vitaorganizer/Texts_de.properties

```diff
@@ -0,0 +1,45 @@
+COLUMN_ICON=Icon
+COLUMN_ID=ID
+COLUMN_PERMISSIONS=Berechtigungen
+COLUMN_SIZE=Gr\u00F6\u00DFe
+COLUMN_TITLE=Titel
+COLUMN_VERSION=Version
+COLUMN_WHERE=Wo
+DELETE_FROM_PSVITA_ACTION=Von der PS Vita l\u00F6schen
+DELETE_FROM_PSVITA_MESSAGE=Bist du sicher, dass du '%title%' l\u00F6schen willst?
+DELETE_FROM_PSVITA_TITLE=Best\u00E4tigung
+INFORMATION=Information
+LOCATION_BOTH=BEIDES
+LOCATION_NONE=KEINES
+LOCATION_PC=PC
+LOCATION_VITA=VITA
+MENU_ABOUT=Über...
+MENU_CHECK_FOR_UPDATES=Nach Aktualisierungen suchen..
+MENU_EXIT=Beenden
+MENU_FILE=Datei
+MENU_HELP=Hilfe
+MENU_REFRESH=Aktualisieren
+MENU_REPACK=Neu verpacken \: Kompressionsstufe 9 + entferne Duplikate + mache es sicher
+MENU_SELECT_FOLDER=Ordner ausw\u00E4hlen...
+MENU_SHOW_EXPLORER=Datei im Explorer anzeigen
+MENU_SHOW_FINDER=Datei im Finder anzeigen
+MENU_SHOW_PSF=PSF anzeigen
+MENU_WEBSITE=\u00D6ffne Webseite
+PERMISSIONS_SECURE=SICHER
+PERMISSIONS_UNSECURE=\!UNSICHER\!
+PSF_VIEWER_TITLE=PSF Betrachter \: %id% \: %title%
+SELECT_PSVITA_VPK_FOLDER=PS VITA VPK Ordner ausw\u00E4hlen
+SEND_DATA_TO_VITA_ACTION=Sende Daten zur PS Vita
+SEND_FULL_APP_TO_VITA_ACTION=Installiere Applikation in nur einem Schritt (erfordert VitaShell >\= 0.9.5)
```

Comment:
"App" is used in german as well and "Applikation" is way longer which may result in linebreaks in the menu.